src/test/resources/com/puppycrawl/tools/checkstyle/checks/finalparameters/InputFinalParametersPrimitiveTypes.java

```diff
@@ -11,9 +11,15 @@
 public class InputFinalParametersPrimitiveTypes
 {
     void foo(int i) {} //no warning
-    void foo1(int i, String k, float s) {}  // violation
-    void foo2(String s, Object o, long l) {}  // 2 violations
-    void foo3(int[] array) {}  // violation
-    void foo4(int i, float x, int[] s) {}  // violation
-    void foo5(int x, long[] l, String s) {}  // 2 violations
+    void foo1(int i, String k, float s) {}  // violation, 'k' should be final
+    void foo2(String s, Object o, long l) {}
+    // 2 violations above
+    // 's' should be final
```

Comment:
Please indent message to +2 spaces as minimal to let it be read  more clearly.
Please put ":" after `above`.
https://github.com/search?q=repo%3Acheckstyle%2Fcheckstyle%20%22violations%20above%22&type=code

Reply:
@romani , Done ✅